arkouda/timeclass.py

```diff
@@ -465,6 +466,9 @@ def to_pandas(self):
         '''
         return to_timedelta(self.to_ndarray())
 
+    def std(self, ddof : Union[int, np.int64] = 0):
+        return Timedelta(ak_array([np.int64(self._data.std(ddof=ddof))]))
```

Comment:
I _think_ you can avoid the ak_array pdarray creation here by doing something like: 
```
return self._scalar_callback(self._data.std(ddof=ddof))
``` 
Since we're only expecting a scalar value (wrapped as a Timedelta) I think that's what you want.

I tested the change and it works with the unit test if you remove the correspnoding `to_pandas()` call.
i.e.
`self.assertEqual(ak_timedel_std, pd_timedel_std)`

Hopefully someone else can weigh in here on which is the right return type.

Reply:
yeah that seems much cleaner. The return type is a pandas.Timedelta object not an arkouda.Timedelta in this case. That is what is being done is Timedelta.sum below so maybe I should do this for consistency. I can't think of a good reason for `std` and `sum` to have different return types

https://github.com/Bears-R-Us/arkouda/blob/eee8dd21d8f91d2861fdd14ded0d9216fdeb6a10/arkouda/timeclass.py#L468-L470

Reply:
I agree that the return type of `std` should be `pd.Timedelta`, like with `sum`.